extension/parquet/column_writer.cpp

```diff
@@ -1265,7 +1265,11 @@ class StringColumnWriter : public BasicColumnWriter {
 
 			if (validity.RowIsValid(vector_index)) {
 				run_length++;
-				const auto &value = strings[vector_index];
+				auto &value = strings[vector_index];
+
+				if (vector.GetType().id() == LogicalTypeId::BIT) {
+					value = string_t(value.GetDataUnsafe() + 1, value.GetSize() - 1); // skip first byte of bitstring
```

Comment:
I wonder if it perhaps makes more sense to convert bits to strings when writing to Parquet?

Alternatively we could leave this unsupported for now as well.

Reply:
Dito, I've taken it out!